frontend/include/chpl/uast/Decl.h

```diff
@@ -48,28 +48,24 @@ class Decl : public AstNode {
 
  private:
 
-  // Use -1 to indicate that there is no such child.
-  int attributesChildNum_;
   Visibility visibility_;
   Linkage linkage_;
   int linkageNameChildNum_;
 
  protected:
-  Decl(AstTag tag, int attributesChildNum, Visibility visibility,
+  Decl(AstTag tag, Visibility visibility,
```

Comment:
Is it deliberate that attributesChildNum was removd from this constructor?

Reply:
yes, because that constructor does not take an `AstList` to indicate children, so the concept of a `attributeChildNum` (or any *childNum) doesn't make sense